app.vue

```diff
@@ -1,5 +1,7 @@
+
 <template>
-  <div>
-    <NuxtWelcome />
-  </div>
-</template>
\ No newline at end of file
+   <landingPage/>
```

Comment:
Then, take a look a the [NuxtPage](https://v3.nuxtjs.org/api/components/nuxt-page) docs.

TL;DR 

You'll need to replace `<landingPage/>` with `<NuxtPage/>`.